packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/use-flow.md

```diff
@@ -1,26 +1,26 @@
+In addition to configuring the Interaction Code grant type in your [authorization server](#set-up-your-authorization-server) and [your app](#enable-interaction-code-grant-on-an-application), make sure you’re using the interaction code flow when you [initialize the Sign-In Widget](/docs/guides/oie-embedded-common-download-setup-app/-/main/#set-up-the-sign-in-widget-and-sdk-for-your-own-app):
 
 * In Okta Sign-In Widget version 7+, Identity Engine is enabled by default, so there are no additional steps to take.
-* If you are using an earlier version than 7, you must explicitly enable Identity Engine features by setting `useInteractionCodeFlow: true` in the config.
+* If you’re using an earlier version than 7, you must explicitly enable Identity Engine features by setting `useInteractionCodeFlow: true` in the config.
 
-> **Note**: Also see [setting up your Okta org](/docs/guides/oie-embedded-common-org-setup/) for typical use case configurations.
+> **Note**: For typical use case configurations, see [setting up your Okta org](/docs/guides/oie-embedded-common-org-setup/).
 
-### Embedded authentication app using a native Identity Engine SDK
+### Apps using an embedded Identity Engine SDK only
 
-If you have a native or web app, use an appropriate [Identity Engine SDK](/docs/guides/oie-embedded-common-download-setup-app/) to start the Interaction Code flow by initializing the authentication client object and then calling the `authenticate()` method corresponding to the SDK that you are using. The remediation process is implemented by handling the authentication status responses of the authentication client object. See how this flow is built by reviewing the [Build a use case with the embedded SDK](/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/) guide. When the remediation process completes successfully, your app is provided with an Interaction Code to then exchange for OpenID Connect tokens to use for authenticated access to your app’s resources.
+If you have a mobile or web app, use an appropriate [Identity Engine SDK](/docs/guides/oie-embedded-common-download-setup-app/) to start the Interaction Code flow by initializing the authentication client object and then calling the `authenticate()` method corresponding to the SDK that you are using. The remediation process is implemented by handling the authentication status responses of the authentication client object. See how this flow is built by reviewing the [Build a use case with the embedded SDK](/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/) guide. When the remediation process succeeds, Identity Engine sends your app an Interaction Code to exchange for OIDC tokens. Your app uses these tokens to access to your app’s resources.
```

Comment:
Please confirm with the fed team if Interaction Code - remediation can be controlled in a mobile app?

Reply:
Checking.

Reply:
@vanngo-okta This is true for all app types. Oddly, the Fed team said they weren't the ones to check with. "All SDK-related questions should be directed to #eng-developer-community-products" apparently.